### PR TITLE
Remove edit/delete/add buttons for hazard actions/strikes in view mode

### PR DESCRIPTION
**Pull request by CarlosFdez**

### CODE REVIEW

File: static/templates/actors/hazard/sheet-new.html [hunk 1]
```diff
@@ -80,7 +80,7 @@ <h3>{{localize "PF2E.HazardResetLabel"}}</h3>
             <div class="section-container">
                 <div class="section-header">
                     <h3>Actions</h3>
-                    {{#if owner}}
```
Comment (stwlam) [inline static/templates/actors/hazard/sheet-new.html:83]: If these `owner` checks are elsewhere, they should be replaced with checking `editable`

File: static/templates/actors/hazard/sheet-new.html [hunk 2]
```diff
@@ -101,7 +101,7 @@ <h4><a>{{action.name}} {{{actionGlyph action.actionCost}}}</a></h4>
                                     </div>
                                     <div class="item-controls">
                                         <a class="item-chat chat" title="{{ localize 'PF2E.NPC.SendToChat'}}"><i class="fas fa-comment-alt"></i></a>
```
Comment (stwlam) [inline static/templates/actors/hazard/sheet-new.html:103]:
```suggestion
                                        <a class="item-chat chat" title="{{localize "PF2E.NPC.SendToChat"}}"><i class="fas fa-comment-alt"></i></a>
```

Comment (CarlosFdez) [reply static/templates/actors/hazard/sheet-new.html:103]: copy paste laziness strikes again

File: static/templates/actors/hazard/sheet-new.html (hunk 1 above)
Comment (CarlosFdez) [reply static/templates/actors/hazard/sheet-new.html:83]: these are the last ones I think unless the editor params count

Comment (stwlam) [reply static/templates/actors/hazard/sheet-new.html:83]: It should just be ensured that nothing is editable when viewed from a locked compendium, on pain of server errors.

**Pull request merged**